src/server.js

```diff
@@ -73,6 +76,13 @@ module.exports = class Server {
       });
   }
 
+  optionsHandler(req, res) {
+    res.set('Access-Control-Allow-Origin', req.get('origin'));
+    res.set('Access-Control-Allow-Credentials', true);
+
+    res.sendStatus(204);
```

Comment:
See https://www.npmjs.com/package/cors#enabling-cors-pre-flight
The options request is needed for preflight requests, for GET request a preflight request is done when there are extra, non common headers, or when cookies are required to be sent (withCredentials).